### DISCUSSION TITLE
[FEATURE]how to switch to Chinese?

### DISCUSSION THREAD

**Discussion by WenjunLiu6146** [crewAIInc/crewAI#1622] [Feature Requests]
### Feature Area

Agent capabilities

### Is your feature request related to a an existing bug? Please link it here.

There is an option in crew : "language", but how to switch the original English to Chinese? I have tried some options but failed. I find [here](https://github.com/crewAIInc/crewAI/blob/main/src/crewai/translations/en.json) has the en.json file. Should I change this so that the language can be swiched?

### Describe the solution you'd like

Solution to use crewai in Chinese.

### Willingness to Contribute

Yes, I'd be happy to submit a pull request

**Comment by 2ndpic** [comment]
如果你发现 AI 仍然有时候回复英文，可以在 Agent 或 Task 里 明确要求使用中文，比如：

```python
研究员 = Agent(
    role="AI 研究员",
    goal="研究人工智能的最新趋势，并撰写一份详细的中文报告。",
    backstory="你是一名专业的人工智能研究员，擅长分析最新的AI技术。所有回答必须使用中文。",
)
```
或者在 Task 里加一句：

```python
研究任务 = Task(
    description="请你研究当前人工智能的最新发展趋势，并总结成一份中文报告。所有回答必须用中文。",
    expected_output="一份详细的中文研究报告，至少 1000 字。",
    agent=研究员,
)
```
✅ 这样，AI 一定会使用中文！

以上是CrewAI Assistant说的，不知道有不有用
